packages/mobile/ios/AudiusReactNative/AppDelegate.m

```diff
@@ -78,6 +78,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
   GCKDiscoveryCriteria *criteria = [[GCKDiscoveryCriteria alloc] initWithApplicationID:@"222B31C8"];
   GCKCastOptions* options = [[GCKCastOptions alloc] initWithDiscoveryCriteria:criteria];
+  // Allow our app to control chromecast volume
+  options.physicalVolumeButtonsWillControlDeviceVolume = YES;
```

Comment:
This won't actually work today: 
https://developers.google.com/cast/docs/ios_sender/integrate#physical_button_volume_control